samples/Echo_Ssl/app/application.cpp

```diff
@@ -109,19 +109,24 @@ bool onReceive(TcpClient& tcpClient, char *data, int size) {
 	return tcpClient.send(data, size);
 }
 
-void connectOk()
+void connectOk(String ssid, uint8_t ssid_len, uint8_t bssid[6], uint8_t channel)
 {
-	debugf("Connected. Got IP: %s", WifiStation.getIP().toString().c_str());
-
-	client = new TcpClient(TcpClientDataDelegate(onReceive));
-	client->addSslOptions(SSL_SERVER_VERIFY_LATER);
-	client->connect(IPAddress(SERVER_IP), 4444, true);
+	debugf("Connected.");
 }
 
-void connectFail()
+void connectFail(String ssid, uint8_t ssid_len, uint8_t bssid[6], uint8_t reason)
 {
 	debugf("I'm NOT CONNECTED!");
-	WifiStation.waitConnection(connectOk, 10, connectFail); // Repeat and check again
+	WifiStation.disconnect();
```

Comment:
This is how it's done in `samples/Websocket_Client`, the only example that uses `WifiEvents` atm. So how this should be handled?